app/views/teams/show.html.erb

```diff
@@ -18,6 +18,33 @@
   <dd><%= @team.description %></dd>
 </dl>
 
+<table class="table table-striped">
+  <thead>
+    <tr>
+    </tr>
+  </thead>
+  <tbody>
+    <% @team.members.each do |member| %>
+      <tr>
+        <td>
+            <%= member.id%> <%= member.first_name%> <%= member.last_name%>
```

Comment:
This and the following lines are indented two spaces too far (file uses two-space indentation).